crates/forge/src/pretty_printing.rs

```diff
@@ -60,19 +64,24 @@ pub(crate) fn print_test_result(test_result: &TestCaseSummary, fuzzer_runs: Opti
         _ => String::new(),
     };
 
-    let fuzzer_runs = match fuzzer_runs {
+    let fuzzer_report = match fuzzer_runs {
         None => String::new(),
         Some(runs) => {
+            let fuzzer_seed_string = match fuzzer_seed {
+                Some(seed) => format!(", fuzzer seed = {seed}"),
+                None => String::new(),
+            };
+
             if matches!(test_result, TestCaseSummary::Failed { .. }) {
                 let arguments = test_result.arguments();
-                format!(" (fuzzer runs = {runs}, arguments = {arguments:?})")
+                format!(" (fuzzer runs = {runs}{fuzzer_seed_string}, arguments = {arguments:?})")
             } else {
-                format!(" (fuzzer runs = {runs})")
+                format!(" (fuzzer runs = {runs}{fuzzer_seed_string})")
             }
         }
     };
 
-    println!("{result_header} {result_name}{fuzzer_runs}{result_message}");
+    println!("{result_header} {result_name}{fuzzer_report}{result_message}");
```

Comment:
I'm not sure if it makes sense to print the seed in case the user configured it manually in the test. I understand that the logic was to emphasise that it is different than the seed output at the end of test run, but not sure if it's necessary.

wdyt @MaksymilianDemitraszek @piotmag769

Reply:
I think it's fine to print it